shared/src/vyxal/parsing/Lexer.scala

```diff
@@ -186,7 +188,8 @@ object Lexer:
 
     // Now, bind the move right tokens to the next token
 
-    var bound = ListBuffer[LitToken | (LitToken, Int)]()
+    val bound: ListBuffer[LitToken | (LitToken, Int)] =
+      ListBuffer[LitToken | (LitToken, Int)]()
```

Comment:
```suggestion
    val bound = ListBuffer[LitToken | (LitToken, Int)]()
```
The type annotation *should* be unnecessary, could you see if this works?